### PR TITLE
moving examples from vault to use-cases section

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Description
Reviewed content in the Vault section and moved examples from there into the use-cases section. Updated screenshots where needed.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by LoryApiFortress** [comment]
@aliciakrebs Please review. Thanks

File: docs/api-testing/variables-environment-overrides.md [hunk 1]
```diff
@@ -69,6 +69,8 @@ The [**Company Vault**](/api-testing/vault/#company-vault) is where you can stor
 
 The [Project **Vault**](/api-testing/vault/#project-vault) is where you can save variables and snippets that will be specific to that Project (i.e., across all tests). Its variables are injected during test execution.
 
+If the same variable exists in both in the **Company Vault** and Project-specific **Vault**, the one used in the **Project Vault** will win over the **Company Vault**.
```
Comment (aliciakrebs) [inline docs/api-testing/variables-environment-overrides.md:72]:
```suggestion
If the same variable exists in both in the **Company Vault** and Project-specific **Vault**, the one used in the **Project Vault** will take precedence over the **Company Vault**.
```

File: docs/api-testing/use-cases/vault-variable.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+id: vault-variable
+title: Saving a Variable in Vault
+sidebar_label: Saving a Variable in Vault
+description: 'Create a variable in the vault to use across your tests'
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Consider a scenario where all the endpoints you have to test requires a Request Header.
```
Comment (aliciakrebs) [inline docs/api-testing/use-cases/vault-variable.md:10]:
```suggestion
Consider a scenario where a Request Header is required for all the endpoints in your test.

File: docs/api-testing/use-cases/vault-variable.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+id: vault-variable
+title: Saving a Variable in Vault
+sidebar_label: Saving a Variable in Vault
+description: 'Create a variable in the vault to use across your tests'
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Consider a scenario where a Request Header is required for all the endpoints in your test.
```
Comment (LoryApiFortress) [inline docs/api-testing/use-cases/vault-variable.md:10]:
```suggestion
Consider a scenario where a Request Header is required for all the endpoints in your test.
```

File: docs/api-testing/variables-environment-overrides.md [hunk 4]
```diff
@@ -79,9 +81,11 @@ If the same variable exists in both in the **Globals/Input Set** and Project-spe
 
 ### Overriding Variables, Environments
 
-When you declare an overridden variable (using the API, [saucectl command line utility](/api-testing/integrations/yaml/#env), or the [**Schedule** tool](/api-testing/schedule-test)) its value will be injected into the test when it's executed. If the variable has already been declared in the **Vault** or the **Globals/Input Set**, it will be rewritten with the new value.
+When you declare an  variable override (using the [API](/dev/api/api-testing/), [saucectl command line utility](/api-testing/integrations/yaml/#env), or the [**Schedule** tool](/api-testing/schedule-test)) its value will be injected into the test when it is executed.
```
Comment (LoryApiFortress) [inline docs/api-testing/variables-environment-overrides.md:84]:
```suggestion
When you declare a variable override (using the [API](/dev/api/api-testing/), [saucectl command line utility](/api-testing/integrations/yaml/#env), or the [**Schedule** tool](/api-testing/schedule-test)) its value will be injected into the test when it is executed.
```

**Pull request merged**